src/graphql/config.js

```diff
@@ -17,4 +17,11 @@ exports.config = {
       url: 'https://api.thegraph.com/subgraphs/name/ianlapham/uniswapv2',
     },
   },
+  arc: {
+    document: './queries/arc.graphql',
+    schema: {
+      method: 'GET',
+      url: 'https://arc-graphql.rainbow.me/graphql',
```

Comment:
Can you set this up to use the staging URL when not in prod? Peep `resources/f2c` for example

Reply:
aye